src/src/AppBundle/Form/RegistrationForm.php

```diff
@@ -23,32 +23,36 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     new NotBlank(),
                     new Length(array('max' => 100)),
                 ),
+                'label'=>$options['data']['lang']['first_name']
```

Comment:
Vsade sa opakuje `$options['data']['lang']` vytiahol by som si to do pomocnej premennej napr. `$labels`.

Reply:
Oki